.github/ISSUE_TEMPLATE/provider-issue.md

```diff
@@ -19,6 +19,10 @@ Please list the resources as a list, for example:
 
 If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.
 
+### Environment variable names
+To get relevant environment variable _names_ please copypaste the output of the following command:
+`$ env | sort | grep -E 'DATABRICKS|AWS|AZURE|ARM|TEST' | awk -F= '{print $1}'`
+
```

Comment:
good call out here

Reply:
yeah, with so many options available we should know which ones are set and possibly causing a conflict.